pkgs/by-name/pt/ptcpdump/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  libpcap,
+}:
+
+buildGoModule (final: {
```

```suggestion
buildGoModule (finalAttr: {
```
finalAttr is the same as the common name